cl_sii/rtc/data_models.py

```diff
@@ -321,6 +351,9 @@ class CesionL0:
         - Same timestamp as the "Registro AoR DTE" event ``DTE Cedido``.
         - The above statements were empirically verified for
           ``CesionNaturalKey(dte_key=DteNaturalKey(Rut('99***140-4'), 33, 3105), seq=2)``.
+        - When receiving an XML AEC document, the SII validates this date is before or
+          equal to the current day.
+          Source: (https://www.sii.cl/factura_electronica/ins_tecnico.pdf)
```

Comment:
Reference a permalink please, and indicate exactly where in that document the rule is (and perhaps even quote some text, if possible)